website/src/pages/en/subgraphs/_meta-titles.json

```diff
@@ -1,6 +1,6 @@
 {
   "querying": "Querying",
   "developing": "Developing",
-  "cookbook": "Cookbook",
+  "guides": "How-to Guides",
```

Comment:
I believe this change has to be applied to all languages or the build will fail.